simple/stats/db.py

```diff
@@ -12,10 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
```

Comment:
should we have a flag to disable this before the cloudsql is supported. Otherwise, custom DC would write to sqlite twice, one from here and one from mixer?

Reply:
Good idea. Done.